installer/kickstart-static64.sh

```diff
@@ -1,9 +1,11 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
the static installer should work on any sh, not just bash.

Reply:
so why are we using things like `local` in it?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2039

Reply:
hm... probably because they work on most `sh` we tested (alpine, old ubuntu, freebsd, macos).

Reply:
btw, we should remove "local" if it is not standard on `sh`. But we should check that there will be no overlapping variables names between functions and global.

Reply:
It is not only `local` if it were, I would remove it. Will investigate later what can be done about this.